website/docs/reference/resource-configs/teradata-configs.md

```diff
@@ -179,7 +186,10 @@ seeds:
     )
   }}
   ```
-  :information_source: Note, unlike in `table_option`, there are no commas between index statements!
+
+  :::info Separators between statements
```

Comment:
Is this a duplication over the above call out?

Reply:
Yes. I've just sent another commit to consolidate into one.